src/main/java/pl/jakubowskir/timetable/trainer/controller/TrainerDto.java

```diff
@@ -0,0 +1,7 @@
+package pl.jakubowskir.timetable.trainer.controller;
+
+public class TrainerDto {
```

Comment:
to może być `record`

Reply:
@Gorgios próbowałem to zrobić, ale miałem problem z getterami, nie wiem trochę czemu. 
Mówiąc szczerze jak to robiłem to myślałem, że wszystko spoko, ale teraz tak na to patrzę i w zasadzie nie mam pojęcia jak to zrobiłem xDD

Reply:
Możesz dać sam record, tam gettery są automatycznie dodane.